### PR TITLE
Correct plan requirements in readme

### PR DESCRIPTION
**Pull request by jrouly**
I have an active Performance plan, and was able to execute CI tasks on a self hosted Kubernetes runner.

The prerequisite that the Scale plan is required is inaccurate.

### CODE REVIEW

**Comment by jrouly** [comment]
cc @MaT1g3R

**Comment by jrouly** [comment]
cc @nramabad @polymeris @sanajaved7 @sulami

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 Repository with various files to install CircleCI's runner on Kubernetes via Helm chart.
 
 ## Prerequisites
-- You must be on our [Scale Plan](https://circleci.com/pricing/) or sign up for a trial. [Reach out to our sales team](https://circleci.com/contact-us/?cloud) to ask about both.
+- You must be on one of our [Performance or Scale plans](https://circleci.com/pricing/), or sign up for a trial. [Reach out to our sales team](https://circleci.com/contact-us/?cloud) to ask about both.
```
Comment (jrahme-cci) [inline README.md:6]: I think runner is available on all plans now, aside from custom?

Comment (jrouly) [reply README.md]: Done. Just removed the plan prerequisite.

**Review by jrahme-cci — APPROVED** [review]

**Pull request merged**